app/views/settings/index.html.erb

```diff
@@ -18,6 +18,8 @@
           <div class="actions">
             <%= f.submit "Update", class: "btn-lg btn-primary btn-block" %>
           </div>
+          <h4 class="text-center">or</h4>
```

Comment:
Thanks for the feedback...

I will check this work and update as necessary...

Have fun,

Joe

On Fri, Apr 17, 2015 at 6:50 AM, Ben Woodall <notifications@github.com>
wrote:

> In app/views/settings/index.html.erb
> <https://github.com/benwoody/shrtnr/pull/16#discussion_r28594936>:
>
> > @@ -18,6 +18,8 @@
> >            <div class="actions">
> >              <%= f.submit "Update", class: "btn-lg btn-primary btn-block" %>
> >            </div>
> > +          <h4 class="text-center">or</h4>
>
> We need a bit more than this, this actually creates an entirely different
> user when you click this button!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/benwoody/shrtnr/pull/16/files#r28594936>.
>



-- 
Joe Simpson
“Reasonable people adapt themselves to the world. Unreasonable people
attempt to adapt the world to themselves. All progress, therefore, depends
on unreasonable people.”
George Bernard Shaw